bl-languages/fi_FI.json

```diff
@@ -0,0 +1,242 @@
+{
+	"language-data":
+	{
+		"native": "Suomi (Suomi)",
+		"english-name": "Finnish",
+		"last-update": "2016-06-11",
+		"author": "Tuomas K.",
+		"email": "",
+		"website": ""
+	},
+
```

Comment:
Great work! :) I made some suggestions for improved translations because I'm such a nitpicker.

Aloitin Bluditin kääntämisen ennen kuin huomasin, että suomenkielinen käännös on jo olemassa, joten ammensin vähän omista käännöspohdinnoista muutosehdotuksia. Jotkut harvat muutokset olivat kielioppiin liittyviä, kun taas suurin osa noteista on ihan vain subjektiivisia mielipiteitä, miten asian voisi ilmaista.

Tagit voisivat olla myös avainsanoja, koska avainsana on söpömpi sana kuin tagi, mutta tagi on toisaalta niin vakiintunut ilmaisu, että en viitsinyt siihen turhaan puuttua käännöksissä.